include/RAJA/policy/openmp/params/kernel_name.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef OPENMP_KERNELNAME_HPP
+#define OPENMP_KERNELNAME_HPP
+
+#include "RAJA/pattern/params/kernel_name.hpp"
+
+namespace RAJA {
+namespace expt {
+namespace detail {
+
+#if defined(RAJA_ENABLE_OPENMP)
+
+  // Init
+  template<typename EXEC_POL>
+  camp::concepts::enable_if< type_traits::is_openmp_policy<EXEC_POL> >
+  init(KernelName&)
+  {
+    //TODO: Define kernel naming
```

Comment:
@artv3 please add issues for things like this so we don't forget to address them later. Thank you.

Reply:
Sure thing! We have it captured here: https://github.com/LLNL/RAJA/issues/1664